### DISCUSSION TITLE
Attempt Block Recovery tied to Additional CSS requires Super Admin for Multisites

### DISCUSSION THREAD

**Discussion by mrdonduck** [WordPress/gutenberg#42078] [Developer Experience]
I believe there is an issue with how core treats the Additional CSS option in the theme customizer. It appears you need the highest level permissions in order to access the Additional CSS, in a single site it makes partial sense an Administrator would only have access to edit CSS, however, in a multisite this has been designated to only Super Admins, not even the subsite Administrator role can access the Additional CSS in the theme customizer.

Because of this permissions disparity for CSS we often have highly stylized blocks that break for anyone editing a page that is of any lesser role than Super Admin. When these blocks break, the user is unable to attempt block recovery and requires a Super Admin in order to recover the block. No other role can successfully recover the block, as a Super Admin role you can edit the page and the blocks that have issues almost never break when updating, for other roles it is 100% of the time that these specific blocks break and are then unrecoverable by the user.

This leads me to believe there is an issue with how blocks and access to the theme's CSS is performed. Obviously giving Super Admin privileges as the only way to resolve this issue is not viable for a Multisite Network and the access should at least include both Super Administrators and Subsite Administrators.

My apologies if this is the wrong place to post this discussion, however, since it involves the block recovery aspect in Gutenberg I thought it made sense here.

**Comment by afragen** [comment]
https://wordpress.org/plugins/unfiltered-mu/ might fix the issue.
